wandb/sdk/service/streams.py

```diff
@@ -55,8 +55,10 @@ class StreamRecord:
     _iface: InterfaceRelay
     _thread: StreamThread
     _settings: SettingsStatic  # TODO(settings) replace SettingsStatic with Setting
+    _active: bool
```

Comment:
nit: do you mind making it some kind of an object, either enum or something else (I assume we will keep other states)

Reply:
I kept it as a boolean as it isnt clear what the other states will be and whether they will be independent..
i did rename it to be more clear, and added an explicit "stream_start" action.